arches/app/models/resource.py

```diff
@@ -189,22 +189,31 @@ def bulk_save(resources):
             resource.tiles = resource.get_flattened_tiles()
             tiles.extend(resource.tiles)
 
-        # need to save the models first before getting the documents for index
+        # need to handle if the bulk load is appending tiles to existing resources/
+        existing_resources = Resource.objects.filter(resourceinstanceid__in=[resource.resourceinstanceid for resource in resources])
+        existing_resources_ids = [existing_resource.resourceinstanceid for existing_resource in existing_resources]
+        resources_to_create = [resource for resource in resources if resource.resourceinstanceid not in existing_resources_ids]
+
         start = time()
-        Resource.objects.bulk_create(resources)
+        Resource.objects.bulk_create(resources_to_create)
         TileModel.objects.bulk_create(tiles)
 
-        print(f"Time to bulk create tiles and resources: {datetime.timedelta(seconds=time() - start)}")
+        logger.info(f"Time to bulk save tiles and resources: {datetime.timedelta(seconds=time() - start)}")
 
         start = time()
-        for resource in resources:
+        for resource in resources_to_create:
```

Comment:
we may also want to save an edit for resources being appended to where`` edit_type="append"`` or something.
This could be done where we iterate through ``existing_resources`` this for loop through something like this:
``for resource in existing_resources:

Reply:
This makes sense so I have added to the branch.